app/models/ability.rb

```diff
@@ -187,6 +187,10 @@ def initialize(user)
       can :toggle_visibility, Organization do |o|
         can? :teach, o
       end
+
+      can :view_custom_points_url, Course do |c|
+        can?(:teach, c) || User.course_students(c).include?(user)
```

Comment:
is the include check done in db or does it load all students to memory and iterate over them?